README.md

```diff
@@ -1,8 +1,10 @@
-# CloudPak Community Automation
+# Redhat Openshift Container (OCP) Community Automation
```

Comment:
I would use the official terms/spacing/case here

# Red Hat OpenShift Container Platform (OCP) Community Automation